webviz_ert/plugins/_response_correlation.py

```diff
@@ -33,31 +33,31 @@ def tour_steps(self) -> List[Dict[str, str]]:
             {
                 "id": self.uuid("response-overview"),
                 "content": (
-                    "Visualization of the currently selected response "
+                    "Visualization of the currently active response "
                     "where by clicking we can select a new index / timestep for correlation analysis."
```

Comment:
where by clicking we can select "and activate" ...

Reply:
good catch, thanks, although i was gonna wait with doing renaming on / around the index, but sure!

Reply:
done